prismic-model/sliceAnalysis.ts

```diff
@@ -78,8 +78,10 @@ async function main() {
     }
   }
 
-  console.info('=== Slice count ==');
-  Array.from(sliceCounter.entries())
+  const slicesArray = Array.from(sliceCounter.entries());
+
+  console.info(`=== Slice count (${slicesArray.length}) ==`);
```

Comment:
Just wanted to have the # of slices listed as well as I'm aiming for a good reduction at the end of all this.